### PR TITLE
Database migration: script generator and checks with setup.py

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**

### CODE REVIEW

File: golem/database/schemas/010_schema.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    > migrator.drop_not_null(model, *field_names)
+    > migrator.add_default(model, field_name, default)
+
+"""
+
+import datetime as dt
+import peewee as pw
+
+try:
+    import playhouse.postgres_ext as pw_pext
```
Comment (etam) [inline golem/database/schemas/010_schema.py:31]: unused import?

File: golem/database/schemas/011_schema.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+except ImportError:
+    pass
+
+
+def migrate(migrator, database, fake=False, **kwargs):
+    """Write your migrations here."""
+
+    migrator.drop_index('hardwarepreset', 'name')
+
+    migrator.add_index('hardwarepreset', 'name', unique=True)
```
Comment (etam) [inline golem/database/schemas/011_schema.py:40]: Why is there dropping and adding index? Rollback seems to do the same.

**Comment by etam** [comment]
There is a migration from 10 to 11, but what are the model changes?

File: golem/database/schemas/011_schema.py (hunk 2 above)
Comment (mfranciszkiewicz) [reply golem/database/schemas/011_schema.py:40]: I see nothing strange in rebuilding indexes here, and peewee_migrate probably does this as a safeguard. In fact, migration / upgrade would a good place to vacuum the database.

File: golem/database/schemas/010_schema.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply golem/database/schemas/010_schema.py:31]: Also, the imports are not organized correctly. This is only one of the issues with scripts generated by peewee_migrate.

**Comment by mfranciszkiewicz** [comment]
Changes are formal, since I’ve introduced 2 new field types for peewee_migrate compat. Db schema does not change.

**Comment by etam** [comment]
The changes in model, I asked for, are in #2015

**Comment by mfranciszkiewicz** [comment]
@etam You already have your answer. Please either approve or reject and specify the reason.

**Review by etam — APPROVED** [review]

**Pull request merged**